CHANGELOG.md

```diff
@@ -1,7 +1,24 @@
-## 0.x.x - Unreleased
+## 0.4.0 - 20 November 2018
 
 ### Added
+* Added support for self-signed certificates and reading from Windows/Mac certificate stores (currently opt-in) [#613](https://github.com/Microsoft/vscode-docker/issues/613), [#602](https://github.com/Microsoft/vscode-docker/issues/602), [#483](https://github.com/Microsoft/vscode-docker/issues/483)
+* Use a different icon for unhealthy containers (thanks @grhm) [#615](https://github.com/Microsoft/vscode-docker/issues/615)
```

Comment:
Might be better to be under a 'Thank you' section?

Reply:
Maybe, but it's the way it's done throughout the rest of the file.